### PR TITLE
[Compute] Disk encryption set: add --enable-auto-key-rotation

### PR DESCRIPTION
**Pull request by qwordy**
**Description**

Resolve https://github.com/Azure/azure-cli/issues/17064
Resolve https://github.com/Azure/azure-cli/issues/17063

New API version 2020-12-01

```
    --enable-auto-key-rotation     : Enable automatic rotation of keys.  Allowed values: false,
                                     true.
```

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

File: src/azure-cli-core/azure/cli/core/profiles/_shared.py [hunk 1]
```diff
@@ -139,8 +139,8 @@ def default_api_version(self):
         ResourceType.MGMT_NETWORK: '2020-11-01',
         ResourceType.MGMT_COMPUTE: SDKProfile('2020-12-01', {
             'resource_skus': '2019-04-01',
-            'disks': '2020-09-30',
-            'disk_encryption_sets': '2020-06-30',
+            'disks': '2020-12-01',
```
Comment (yungezz) [inline src/azure-cli-core/azure/cli/core/profiles/_shared.py:142]: re-recording tests?

Comment (qwordy) [reply src/azure-cli-core/azure/cli/core/profiles/_shared.py:142]: No SDK now. I wrote this PR in advance.

File: src/azure-cli/requirements.py3.Linux.txt [hunk 2]
```diff
@@ -29,7 +29,7 @@ azure-mgmt-billing==1.0.0
 azure-mgmt-botservice==0.3.0
 azure-mgmt-cdn==11.0.0
 azure-mgmt-cognitiveservices==6.3.0
-azure-mgmt-compute==19.0.0
+azure-mgmt-compute==20.0.0
```
[the remark this answers is not in the document]
Comment (qwordy) [reply src/azure-cli/requirements.py3.Linux.txt:32]: No. Just disk.

**Review by kairu-ms — APPROVED** [review]

**Review by yungezz — APPROVED** [review]

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**